test/test-app/ts_upgrade/src/app/strict-null-checks/strict_null_checks_component.ts

```diff
@@ -0,0 +1,168 @@
+/*
```

Comment:
I would assume all file under test-app/ts_upgrade are just golden file, let me know if it is not.

Reply:
Oops, didn't mean to commit the golden files in, I'll remove it from the cl.